### PR TITLE
Rescu

### PR DESCRIPTION
**Pull request by jovenc**
## What this pull request accomplishes:
- Script to read files and insert data into postgresql table
- Deploy the script on terminal server and schedule it to run daily
- Airflow the daily process of checking if the process is running and if the counts make sense
- Added a short description about RESCU in the root README 

## Issue(s) this solves:

- #291 
- #296 

## What, in particular, needs to reviewed:

- The rescu folder and root README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -297,3 +299,17 @@ locations_zone|coordinate of school|text
 final_sign_installation|final sign installation date|text
 locations_fb|location of flashing beacon|text
 locations_wyss|location of watch your speed sign|text
+
+## RESCU - Traffic Camera
```
Comment (radumas) [inline README.md:303]: I am reasonably certain the volumes come from loop detectors

File: dags/check_rescu.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            """.format(
+            task=context.get('task_instance').task_id,
+            dag=context.get('task_instance').dag_id,
+            ti=context.get('task_instance'),
+            exec_date=context.get('execution_date'),
+            log_url=context.get('task_instance').log_url,
+        )
+    failed_alert = SlackWebhookOperator(
+        task_id='slack_test',
+        http_conn_id='slack',
```
Comment (radumas) [inline dags/check_rescu.py:54]: Now seems like a good time to request using the new style of Slack message (see documentation). The people to tag are myself, Charles, and Aakash.

**Review by radumas — CHANGES_REQUESTED** [review]
My bad. I should've stated that `rescu` is a subset of `volume/` data. 

The `rescu` folder needs a README, which should include details on how the python script is run, why there's a dag to check that the data is ok. 

I need to also check that I've committed the changes I made to make it run on the terminal server 😬

**Comment by jovenc** [comment]
RESCU PR comments addressed

File: rescu/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+|volume_uid|detector_id|datetime_bin|volume_15min|arterycode|
+|--|--|--|--|--|
+|9333459|DE0010DEG|2020-01-13 17:00:00|	749|	23984|
+|9333460|DE0010DER|2020-01-13 17:00:00|	80|	3272|
+
+## `check_rescu.py`
+
+Since the terminal server does not have an internet connection, we will not get notified if the process fails. Therefore, we created an Airflow task to do that job for us. There is a dag named [`check_rescu`](/dags/check_rescu.py) which runs at 6am everyday that checks the number of rows inserted for the day before in both tables `rescu.raw_15min` and `rescu.volumes_15min`. If the number of rows is 0 OR if the number of rows from the raw table is less than that in the volumes table OR if the total number of rows from the volumes table is less than 7000 (the average number of rows per day is about 20,000), a Slack notification will be sent to notify the team. The line that does exactly that is shown below.
+```sql
```
Comment (radumas) [inline rescu/README.md:30]: `sql` code block but the code is actually Python.

**Review by radumas — APPROVED** [review]

**Pull request merged**